st2actions/st2actions/runners/actionchainrunner.py

```diff
@@ -374,7 +374,10 @@ def _run_action(self, liveaction, wait_for_completion=True):
             liveaction, _ = action_service.request(liveaction)
         except:
             liveaction.status = LIVEACTION_STATUS_FAILED
-            raise Exception('Failed to schedule liveaction.')
+
+            message = 'Failed to schedule liveaction.'
+            LOG.exception(message)
```

Comment:
well this lead to 2 back to back logs. not a deal breaker. I would prefer not to do that but it is a fairly minor point.